CMakeLists.txt

```diff
@@ -684,6 +693,34 @@ function(AddApplication)
     set_property(TARGET ${A_Target} APPEND PROPERTY COMPILE_DEFINITIONS ${A_Definitions})
     set_target_properties(${A_Target} PROPERTIES OUTPUT_NAME "${A_ExecutableName}" PREFIX "" FOLDER "engine")
 
+    # Append Windows specific manifests.
+    if (WIN32)
+      set(WINDOWS_MANIFESTS_DIR ${ENGINE_DIR}/sys/manifests)
+      target_sources(${A_Target} PRIVATE
+          # App is DPI aware, so Windows doesn't try to scale UI.
+          ${WINDOWS_MANIFESTS_DIR}/dpi-aware.manifest
```

Comment:
So we could remove this part: https://github.com/DaemonEngine/Daemon/blob/63ef00aa7b9bbf388a015cd9ffffc6d09a6429f5/src/engine/framework/System.cpp#L554

Reply:
Done, thanks!